content/slides/workshop-overview.md

```diff
@@ -128,8 +132,6 @@ behaviors in the "Our Standards" section.*
 :::
 
 
-{{< include /content/group-work-expectations.md >}}
```

Comment:
I think we decided to move group roles, not the whole expectations page. If we want to use the include mechanism for that, we'd have to break that page into two files. I still think it would be useful to touch on group work expectations (1 slide) in the overview deck.

Reply:
Decided to just duplicate the group work expectations in the overviews slide. Not a bad idea to get that back in front of them in the introductions slide.

Reply:
8b4a8f7 * origin/trey-overview-todos Add group work expectations slide to workshop overview

Reply:
Are you in any way opposed to using an include to avoid duplication? If not, I'll be happy to make that change.

Reply:
1fc0aec